docs/dev/api/README.md

```diff
@@ -0,0 +1,33 @@
+# Kyverno API
+
+This document provides guidance on extending and maintaining the [Kyverno API](../../../api/kyverno/)
+
+## Concepts
+* https://kubernetes.io/docs/reference/using-api/api-concepts/ 
+* https://kubernetes.io/docs/reference/using-api/ 
+* https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/
+
+## API Groups 
+
+All Kyverno resources are currently defined in the `kyverno.io` API group. 
+
+## API Versions
+
+This `kyverno.io` has the following versions:
+* v1apha1 (deprecated) 
```

Comment:
We @eddycharly  are adding the new cleanup policy using `v1alpha1`, why deprecate it?